src/module/canvas/ruler.ts

```diff
@@ -148,13 +148,20 @@ class RulerPF2e<TToken extends TokenPF2e | null = TokenPF2e | null> extends Rule
                 .filter(
                     (r) =>
                         r.document.behaviors.some(
-                            (b) => !b.disabled && b.type === "environmentFeature" && b.system.terrain.difficult > 0,
+                            (b) =>
+                                !b.disabled &&
+                                b.type === "environmentFeature" &&
+                                b.active &&
+                                b.system.terrain.difficult > 0,
```

Comment:
?

Reply:
Ah, I hadn't noticed that when going through all the cases of checks that matched "environment

Reply:
I will update my PR to only add !b.disabled in places where it is missing